### PR TITLE
Camshaft 0.50.0

### PR DESCRIPTION
**Pull request by ethervoid**
Update to [camshaft 0.50.0](https://github.com/CartoDB/camshaft/releases/tag/0.50.0)

### CODE REVIEW

**Comment by ethervoid** [comment]
Please, @dgaubert or @rochoa could you please take a look at this PR. I've deployed in staging and works without any problem but I want to be sure everything is ok.

**Comment by dgaubert** [comment]
About promise issue in `turbo-carto` adapter,  I'm not sure why we weren't able to detect it before. Promises in `turbo-carto` are there since the beginning of `turbo-carto`. @rochoa, do you know why it throws error now?

**Comment by rochoa** [comment]
It is explained in the cherry-picked commit.

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   ],
   "dependencies": {
     "body-parser": "~1.14.0",
-    "camshaft": "0.48.5",
+    "camshaft": "0.50.0",
     "cartodb-psql": "~0.6.1",
```
Comment (rochoa) [inline package.json]: I would upgrade cartodb-psql to 0.7.1 here or I would keep this version in the shrinkwrap.

File: npm-shrinkwrap.json [hunk 2]
```diff
@@ -238,27 +238,136 @@
               "resolved": "https://registry.npmjs.org/moment/-/moment-2.17.1.tgz"
             }
           }
+        },
+        "cartodb-psql": {
```
Comment (rochoa) [inline npm-shrinkwrap.json]: As I already pointed, I would upgrade cartodb-psql or remove this. That way we only rely on one version of cartodb-psql.

File: package.json (hunk 1 above)
Comment (ethervoid) [inline package.json]: Ouch! on it!

**Comment by dgaubert** [comment]
Oh my.. !! I read that comment like three times but I wasn't able to understand it (Friday afternoon effect). Thanks @rochoa and sorry for the noise.

File: package.json (hunk 1 above)
Comment (ethervoid) [inline package.json]: One question, that means to upgrade [windshaft](https://github.com/CartoDB/Windshaft/blob/master/package.json#L26) dependency too?

File: .gitignore [hunk 3]
```diff
@@ -3,6 +3,7 @@ config.status*
 config/environments/*.js
 .idea
 .vscode
+.nvmrc
```
Comment (ethervoid) [inline .gitignore]: I've added this to `.gitignore` to avoid upload it by mistake

**Comment by ethervoid** [comment]
Ok, ready to deploy :)

**Comment by dgaubert** [comment]
🇨🇭

**Pull request merged**